config.go

```diff
@@ -16,6 +16,17 @@ type cors struct {
 	preflightHeaders http.Header
 }
 
+var (
+	AllowedSchemas = []string{
+		"http://",
+		"https://",
+		"chrome-extension://",
```

Comment:
Please add a new flag to extend another schema like chrome-extension and default only allow `http://` or `https://`.